packages/cli/src/commands/source/push/add.ts

```diff
@@ -185,19 +188,19 @@ export default class SourcePushAdd extends Command {
     fileNames: string[],
     accumulator = this.accumulator
   ) {
-    const send = async (documentBuilders: DocumentBuilder[]) => {
+    // TODO: rename to something more meaningful (e.g. chunksToSend)
```

Comment:
Why is this left as a todo ? Can you rename in this PR ?

Reply:
Sorry. I was supposed to do it and forgot